src/Objects/EventSubSignature.php

```diff
@@ -54,15 +49,11 @@ private static function getTimestamp(?string $rawTimestamp): ?int
             return null;
         }
 
-        if ( ! preg_match(self::TIMESTAMP_PATTERN, $rawTimestamp, $matches)) {
-            return null;
+        if (preg_match('/\.[0-9]{9}Z$/', $rawTimestamp)) {
+            $rawTimestamp = substr_replace($rawTimestamp, '', -2, 1);
         }
 
-        $dateTime = DateTime::createFromFormat(
-            'Y-m-d\TH:i:s.u\Z',
-            sprintf('%s.%dZ', $matches['pre'], substr($matches['nano'], 0, 6))
-        );
-
-        return $dateTime->getTimestamp();
+        $timestamp = strtotime($rawTimestamp);
+        return $timestamp ?: null;
```

Comment:
```suggestion
        return strtotime($rawTimestamp) ?: null;
```

The `timestamp` variable is not needed